lib/cosmos/tools/launcher/launcher_config.rb

```diff
@@ -250,7 +241,5 @@ def parse_launch_gem(command)
 
       raise "Could not find gem containing tool: #{split[1]} - Make sure the appropriate gem is in your Gemfile"
     end
-
-  end # class LauncherConfig
-
-end # module Cosmos
```

Comment:
Why do you keep removing these end comments?

Reply:
They aren't really helpful and they conflict with the rubocop standards